website/src/content/main-concepts/data-fetching.mdx

```diff
@@ -4,11 +4,17 @@ title: Data Fetching
 
 In client-side applications, data fetching is not always easy and it usually involves a lot of boilerplate code around implementation, state management, data normalization, etc.
 
-Since the commercetools platform has first-class support for GraphQL, in Merchant Center we prefer and recommend to use GraphQL as the main data fetching choice.
+Since the commercetools platform has first-class support for GraphQL, we recommend to build Custom Applications using GraphQL as the main data fetching choice.
 
-# GraphQL endpoints
+# Requests to GraphQL APIs
 
-To handle requests to the [GraphQL APIs](/main-concepts/graphql) we use the [Apollo GraphQL Client](https://www.apollographql.com/docs/react/), which comes already configured within the [`<ApplicationShell>`](/main-concepts/application-shell).
+To handle requests to the [GraphQL APIs](/main-concepts/graphql) we use the built-in [Apollo GraphQL Client](https://www.apollographql.com/docs/react/).
+
+<Info>
+
+The Apollo Client is already pre-configured within the [`<ApplicationShell>`](/main-concepts/application-shell) to connect to the `/graphql` endpoint of the [Merchant Center API Gateway](/main-concepts/api-gatway). You don't need to configure it on your own, unless you need to connect to an **external** GraphQL API (see [#connecting-to-an-external-graphql-api]).
```

Comment:
Link **#connecting-to-an-external-graphql-api** is not working.

Reply:
Ups, my bad. Thanks!